src/components/Place/PlaceStats/PlaceStats.tsx

```diff
@@ -23,9 +26,27 @@ export type PlaceStatsProps = {
 export default React.memo(function PlaceStats(props: PlaceStatsProps) {
   const { place, loading, poi, hideUserCount } = props
   const l = useFormatMessage()
+  const [ff] = useFeatureFlagContext()
+  const rating = useMemo(() => {
+    if (
+      !place ||
+      (place.content_rating !== SceneContentRating.ADULT &&
+        place.content_rating !== SceneContentRating.RESTRICTED)
+    ) {
+      return SceneContentRating.TEEN.toLocaleLowerCase()
+    }
+    return place.content_rating.toLocaleLowerCase()
+  }, [place])
```

Comment:
Try abstracting this functionality in a function `getPlaceContentRating(place?: PlacesAttributes): SceneContentRating` and using it here

```tsx
  const rating = useMemo(() => getPlaceContentRating(place), [place])
```